committee/committee.go

```diff
@@ -146,6 +148,10 @@ func NewCommittee(kvstore db.KVStore, cfg Config) (Committee, error) {
 	if cfg.NumOfFetchInParallel > 0 {
 		fetchInParallel = cfg.NumOfFetchInParallel
 	}
+	gravityChainSize := uint64(10)
```

Comment:
I think you want to make it to 1000.

Here 1000 is the eth block gap, and every 100 eth block gap, we will sample election data once, thus resulting into 10 election data points. cc @CoderZhi 

Can you experiment again with 1000? How long does a node need to start?